scripts/travis-build.bash

```diff
@@ -195,7 +195,13 @@ function main () {
     fi
 
     msg "running tests"
-    if ! npm test; then
+    if ! npm run test; then
```

Comment:
"run" is unnecessary.

Reply:
I know, but that's a special case and I like the consistency